frontend/src/shared/components/tasque-task-editing/task-editing.component.html

```diff
@@ -0,0 +1,153 @@
+          <span class="summary-text" [innerHTML]="safeHTML(task.summary)"> </span>
+          <tasque-button
+            class="btn icon edit"
+            [icon]="editIcon"
+            (click)="summaryClick()"
+          ></tasque-button>
+        </div>
+
+        <div
+          class="input-wrapper"
+          *ngIf="this.taskSummaryInputShow"
+          (clickOutside)="summaryInputOutsideClick()"
+          [delayClickOutsideInit]="true"
+        >
+          <tasque-input
+            placeholder="Type some text"
+            formControlName="taskSummary"
+            errorMessage=""
+            [invalid]="!editTaskForm.controls.taskSummary.valid"
+          >
+          </tasque-input>
+        </div>
+
+        <div class="task-summary-buttons">
+          <tasque-button class="btn icon pink" [icon]="linkIcon"></tasque-button>
+          <tasque-button class="btn icon pink" [icon]="paperClipIcon"></tasque-button>
+          <tasque-button class="btn icon pink" [icon]="checkIcon"></tasque-button>
+          <tasque-button class="btn icon pink" [icon]="shareIcon"></tasque-button>
+          <tasque-button class="btn icon" [icon]="ellipsisIcon"></tasque-button>
+        </div>
+      </h5>
+      <span class="error" *ngIf="!this.editTaskForm.controls.taskSummary.valid">{{
+        summaryErrorMessage
+      }}</span>
+    </div>
+
+    <div class="task-main-container">
+      <div class="task-info-container">
+        <div class="item-wrapper">
+          <span class="item-label">Project</span>
+          <tasque-dropdown
+            width="200px"
+            [options]="projectOptions"
+            formControlName="taskProject"
+          ></tasque-dropdown>
+        </div>
+
+        <div class="item-wrapper">
+          <span class="item-label">Status</span>
+          <tasque-dropdown
+            width="200px"
+            [options]="taskStatusOptions"
+            formControlName="taskStatus"
+          ></tasque-dropdown>
+        </div>
+
+        <div class="item-wrapper">
+          <span class="item-label">Priority</span>
+          <tasque-dropdown
+            width="200px"
+            [options]="taskPriorityOptions"
+            formControlName="taskPriority"
+          ></tasque-dropdown>
+        </div>
+
+        <div class="item-wrapper">
+          <span class="item-label">Sprint</span>
+          <tasque-dropdown
+            width="80px"
+            optionsWidth="max-content"
+            [options]="sprintOptions"
+            formControlName="taskSprint"
+          ></tasque-dropdown>
+        </div>
+
+        <div class="item-wrapper">
+          <span class="item-label">Assignee</span>
+          <span formControlName="taskAssignees" ngDefaultControl>User Select Button</span>
+        </div>
+        <div class="item-wrapper">
+          <span class="item-label">Reporter</span>
+          <span>avatar__</span>
+          <span> {{ taskUser.name }}</span>
```

Comment:
I think the profile name should redirect user to the profile page

Reply:
No, i think it shouldn't. Reporter is just a current user that changes current task. There is no need to redirect user to his profile page. There are another way to do it. For example, using dropdown menu from header.